src/usr/local/www/diag_ping.php

```diff
@@ -50,8 +49,8 @@
 	$reqdfieldsn = array(gettext("Host"), gettext("Count"));
 	do_input_validation($_REQUEST, $reqdfields, $reqdfieldsn, $input_errors);
 
-	if (($_REQUEST['count'] < 1) || ($_REQUEST['count'] > MAX_COUNT)) {
-		$input_errors[] = sprintf(gettext("Count must be between 1 and %s"), MAX_COUNT);
+	if (($_REQUEST['count'] < 1) || ($_REQUEST['count'] > MAX_PING_COUNT)) {
```

Comment:
In the context of the ping page, MAX_ & DEFAULT_COUNT constants may be explicit enough. But since now they can be included from another page, renamed them to clearly state they are related to ping actions and potentially avoid collisions.